src/resolve/resolved-conf.c

```diff
@@ -414,3 +414,46 @@ int manager_parse_config_file(Manager *m) {
         return 0;
 
 }
+
+int config_parse_refuse_record_types(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        Manager *m = ASSERT_PTR(userdata);
+        int r;
+        Set *refused_records = NULL;
+
+        refused_records = set_free(refused_records);
+
```

Comment:
in our codebase, if you assign an empty string to a setting it should generally reset the field to its default. Hence, please add:

```c
if (isempty(rvalue)) {
       m->refuse_record_types = set_free(m->refuse_record_types);
       return 0;
```

This is nice because it allows that configuration file drop-ins can override each other and the main config file.